ceph_cfg/ops_cluster.py

```diff
@@ -58,3 +58,73 @@ def status_refresh(self):
                         output["stderr"])
                         )
         self.model.cluster_status = json.loads(output["stdout"].strip())
+
+
+    def _df_node_to_osd_dict(self, node):
+        node_type = node.get("type")
+        if node_type != "osd":
+            return None
+        osd_id = node.get("id")
+        osd_name = node.get("name")
+        osd_weight_crush = node.get("crush_weight")
+        osd_weight = node.get("reweight")
+        osd_kb_total = node.get("kb")
+        osd_kb_used = node.get("kb_used")
+        osd_kb_avail = node.get("kb_avail")
+        osd_pgs = node.get("pgs")
+        osd_utilization = node.get("utilization")
+        return {
+            "id" : osd_id,
```

Comment:
I'd think that sticking to the ceph naming would probably be easier to understand, even if it is not the naming I'd have chosen (because then we're consistent with how ceph calls things and users don't have to remember multiple naming schemes).